explorer/README.md

```diff
@@ -134,16 +133,29 @@ To explain this boilerplate:
 Please refer to
 [Fuzzer documentation](https://github.com/carbon-language/carbon-lang/blob/trunk/explorer/fuzzing/README.md).
 
-## Trace Program Execution
+## Explorer's Trace Output
 
-When tracing is turned on (using the `--trace_file=...` option or `.verbose`
-target), `explorer` prints the state of the program and each step that is
-performed during execution.
+Explorer's Trace Output refers to a detailed record of program phases and their
+internal processes a program goes through when executed using the `explorer`, it
+also records things like changes in memory and action stack that describes the
+state of the program.
+
+Tracing can be turned on (using the `--trace_file=...` option or `.verbose`
+target).
```

Comment:
```suggestion
Tracing can be turned on using the `--trace_file=...` option. Explorer tests
can be run with tracing enabled by using the `<testname>.verbose` test target.
```

I think it's better to list the user-facing trace support first, with the unit tests left as more of an afterthought.